fixtures/jiechuang/54led-3w-rgbw.json

```
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "54LED 3W RGBW",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["CHOI"],
+    "createDate": "2024-06-24",
+    "lastModifyDate": "2024-06-24"
+  },
+  "links": {
+    "productPage": [
+      "https://korean.alibaba.com/p-detail/54led-60704286784.html?spm=a2700.galleryofferlist.normal_offer.d_image.5ae629ae7GPnQn"
+    ]
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe Speed": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speedStart": "1Hz",
+        "speedEnd": "255Hz"
```

```suggestion
        "speedStart": "slow",
        "speedEnd": "fast"
```

I don't have a manual, but I doubt it's 1-255Hz.

Is there a range of values here which disables the strobe, or is one of the "No function" channels a Shutter/Strobe channel?